state/allcollections.go

```diff
@@ -400,12 +398,6 @@ func allCollections() CollectionSchema {
 			}},
 		},
 
-		// This collection holds information about cloud image metadata.
-		cloudimagemetadataC: {
```

Comment:
I think you didn't remove the constant itself here, its in this file.

It has no usages though so I guess you've got the rest :)

Reply:
Nice spot, thanks !